includes/minishell.h

```diff
@@ -1,11 +1,25 @@
 #ifndef MINISHELL_H
 # define MINISHELL_H
 
+// You can delete that comment once you read it.
+// Idea is that with this you can write stuff like DEBUG(print_tokens(get_minishell(NULL));) and that line will only be happening if you "make debug".
```

Comment:
sooo you're saying that while debugging, we have to write DEBUG(xxxxx) somewhere in the code, and that will call that function? But then why not just calling that function? I'm not understanding the applicability/where/how you use this DEBUG macro

Reply:
So, the way I see it, when you call a debugging function and print stuff in the middle of what your code shows, it's super messy.
If you call debug, you can have the messy version. Then half a second later you can simply make and see what your code does.
You also can leave a 'print tokens' where it is without commenting it etc for all future times when you have to debug while working on the project (which with 2 people, plus something of this scale, will happen). You'll see the debugging info only if you want, and you can continue working the rest of the time while not having to remove everything all the time.
I don't know for sure that it will be useful but I sense it will be so I put that in for now.
If it turns out to be useless, we can get rid of it easily

Reply:
Ok, got it